test/monitoringe2e/helpers.go

```diff
@@ -41,6 +41,15 @@ var (
 		"mt_broker_ingress_go_mallocs",
 		"sugar_controller_go_mallocs",
 	}
+
+	serverlessComponentQueries = []string{
+		// Checks if openshift-knative-operator metrics are served
+		"knative_operator_go_mallocs",
+		// Checks if knative-opeshift metrics are served
+		"controller_runtime_active_workers{controller=\"knativeserving-controller\"}",
```

Comment:
We implicitly check this via the knative_up metric elsewhere but does not hurt to verify here.